src/info/info.h

```diff
@@ -26,29 +26,35 @@ Constants
 #define INFO_KEY_VERSION                                            "backrest-version"
     STRING_DECLARE(INFO_KEY_FORMAT_STR);
 
+/***********************************************************************************************************************************
+Function types for loading and saving
+***********************************************************************************************************************************/
+typedef bool InfoLoadCallback(void *data, unsigned int try);
```

Comment:
```
// The purpose of this callback is to attempt a load (from file or otherwise).  Return true when the load is successful or throw an
// error.  Return false when there are no more loads to try, but always make at least one load attempt.  The try parameter will
// start at 0 and be incremented on each call.
```

Reply:
Looks good, thanks.